packages/docusaurus-theme-common/src/utils/docsUtils.tsx

```diff
@@ -183,3 +188,32 @@ export function isActiveSidebarItem(
 
   return false;
 }
+
+export function useDocsBreadcrumbs(): BreadcrumbItem[] {
```

Comment:
I'd rather have this method return something PropCategory / PropDoc instead of a normalized structure like BreadcrumbItem

We want users to be able to customize the look and feel of this breadcrumb with proprietary config, like adding icons for categories or whatever, so it's rather handle all the different sidebar item types directly in the component that renders instead of losing the `type` attribute at this stage